src/views/PendingReviewDetail.vue

```diff
@@ -413,6 +415,11 @@ async function addProduct() {
 }
 
 async function updateItemStatus(statusId: string, item?: any) {
+  if (item) {
```

Comment:
We can move this logic inside the below `if` condition where we are adding items in the list.